javascript-practice/src/styles/bases/_base.scss

```diff
@@ -1 +1,22 @@
 // TODO
+.overlay {
+  @extend %d-absolute;
+
+  height: 100%;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.3);
+  z-index: 2;
+  opacity: 0;
+  pointer-events: none;
+
+  &.active {
+    @extend %active;
+  }
+}
+
+.dark-overlay {
+  @extend .overlay;
+
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 3;
```

Comment:
Pls set variable for index, so we can ez manage it. Btw, consider create level for index

Reply:
Fixed.